pymllm/orchestrator/scheduler_process.py

```
@@ -505,30 +520,40 @@ def event_loop(self) -> None:
         Supports two modes:
         1. Legacy ZMQ: Uses ``zmq.Poller`` with a short timeout
         2. Shared queue: Non-blocking get from multiprocessing.Queue
 
+        When *brief_poll* is ``True`` (typically during active decode), the
+        first poll uses a small timeout (``_DECODE_POLL_TIMEOUT_MS``) instead
+        of zero.  This yields the CPU core to the OS scheduler between decode
+        batches while adding negligible latency.
+
         Messages are either:
         * A :class:`~pymllm.engine.io_struct.TokenizedGenerateReqInput`
           dataclass – appended to ``_waiting_queue``.
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)